parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -72,7 +72,9 @@ if [[ "${SHOULD_CONFIGURE_CUSTOM_CA_TRUST}" == "true" ]]; then
     logs_to_events "AKS.CSE.configureCustomCaCertificate" configureCustomCaCertificate || exit $ERR_UPDATE_CA_CERTS
 fi
 
-registry_domain_name="${MCR_REPOSITORY_BASE%/}"
+registry_domain_name="${MCR_REPOSITORY_BASE:-mcr.microsoft.com}"
```

Comment:
maybe more (re)usable if we default this directly in `GetMCRRepositoryBase` in the func map in baker.go?